views/layout.pug

```diff
@@ -17,17 +17,16 @@ head
   script(src="//code.jquery.com/jquery-2.1.1.min.js")
   script
     = ctx?"window.prismic = { endpoint: '" + ctx.endpoint + "' };":""
-  script(type="text/javascript").
-    window.PrismicProfiles = !{JSON.stringify(PrismicProfiles)}
   script(src="//static.cdn.prismic.io/prismic.min.js")
   script(src='/javascript/lightSlider/js/lightslider.js')
   script(src='/javascript/cookies.js')
```

Comment:
Do we need cookies.js anymore? Can this library be removed now that you've removed the profiles?